### PR TITLE
Update changelog for CameraAnimators (#217)

### PR DESCRIPTION
**Pull request by jmkiley**
PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-ios/blob/main/CONTRIBUTING.md#contributor-license-agreement).

Fixes: < Link to related issues that will be fixed by this pull request, if they exist >

## Pull request checklist:
 - [ ] Briefly describe the changes in this PR.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [ ] Apply changelog label ('breaking change', 'bug :beetle:', 'build', 'docs', 'feature :green_apple:', 'performance :zap:', 'testing :100:') or use the label 'skip changelog'
 - [ ] Add an entry inside this element for inclusion in the `mapbox-maps-ios` changelog: `<changelog></changelog>`.
 - [ ] Update the migration guide, API Docs, Markdown files - Readme, Developing, etc

### Summary of changes

This PR adds changelog entries for the work in #217.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
+* `UIView.animate` is no longer supported. Instead, use `CameraAnimators`. ([#217](https://github.com/mapbox/mapbox-maps-ios/pull/217))
+* Developers should make camera changes directly to `MapView`'s camera properties. Previously, changes could be applied to `MapView.cameraView`. ([#217](https://github.com/mapbox/mapbox-maps-ios/pull/217))
+* `CameraAnimator` objects are managed by developers and should be stored in by developers to prevent the animations from falling out of scope. 
+
+## Features ✨ and improvements 🏁
+* Introduced the platform-driven Drag API for shifting the map’s camera.  ([#217](https://github.com/mapbox/mapbox-maps-ios/pull/217))
+* Introduced `CameraAnimator`, a UIViewPropertyAnimator-based class for animating camera changes. These animators should be created using `CameraManager.makeCameraAnimator` methods. ([#217](https://github.com/mapbox/mapbox-maps-ios/pull/217))
+* Gesture-driven camera changes have been updated to use camera animators. ([#217](https://github.com/mapbox/mapbox-maps-ios/pull/217))
+* The `AnimatorOwner` enum has been added to track owners for individual animators. ([#217](https://github.com/mapbox/mapbox-maps-ios/pull/217))
+
```
Comment (neelmistry94) [inline CHANGELOG.md:15]: Add a line about FlyTo animation is now backed by `CameraAnimators` and handles the following keyframes: `zoom`, `pitch`, `bearing`, and `centerCoordinate`

Comment (jmkiley) [reply CHANGELOG.md:16]: Updated!

**Review by neelmistry94 — APPROVED** [review]
lgtm

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,19 @@
...
 Mapbox welcomes participation and contributions from everyone.
+## Breaking changes ⚠️
+* The `CameraManager.moveCamera` method has been removed. ([#217](https://github.com/mapbox/mapbox-maps-ios/pull/217))
+* `UIView.animate` is no longer supported. Instead, use `CameraAnimators`. ([#217](https://github.com/mapbox/mapbox-maps-ios/pull/217))
+* Developers should make camera changes directly to `MapView`'s camera properties. Previously, changes could be applied to `MapView.cameraView`. ([#217](https://github.com/mapbox/mapbox-maps-ios/pull/217))
+* `CameraAnimator` objects are managed by developers and should be stored in by developers to prevent the animations from falling out of scope. 
+
+## Features ✨ and improvements 🏁
+* Introduced the platform-driven Drag API for shifting the map’s camera.  ([#217](https://github.com/mapbox/mapbox-maps-ios/pull/217))
+* Introduced `CameraAnimator`, a UIViewPropertyAnimator-based class for animating camera changes. These animators should be created using `CameraManager.makeCameraAnimator` methods. ([#217](https://github.com/mapbox/mapbox-maps-ios/pull/217))
```
Comment (julianrex) [inline CHANGELOG.md:12]: This is listed twice, can we consolidate? Can we also link to the migration guide - that will need a section on this too.

Comment (julianrex) [inline CHANGELOG.md:12]: I think we should drop `UIViewPropertyAnimator` here, as that's an implementation detail. We could say UIViewPropertyAnimator-like?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,19 @@
 # Changelog for Mapbox Maps SDK v10 for iOS
 
 Mapbox welcomes participation and contributions from everyone.
+## Breaking changes ⚠️
+* The `CameraManager.moveCamera` method has been removed. ([#217](https://github.com/mapbox/mapbox-maps-ios/pull/217))
+* `UIView.animate` is no longer supported. Instead, use `CameraAnimators`. ([#217](https://github.com/mapbox/mapbox-maps-ios/pull/217))
+* Developers should make camera changes directly to `MapView`'s camera properties. Previously, changes could be applied to `MapView.cameraView`. ([#217](https://github.com/mapbox/mapbox-maps-ios/pull/217))
```
Comment (julianrex) [inline CHANGELOG.md:7]: Is cameraView now internal?

**Pull request merged**